fenix/app/src/main/res/values/strings.xml

```diff
@@ -285,6 +285,9 @@
     <!-- Title for set firefox as default browser screen.
         The first parameter is the name of the app defined in app_name (for example: Fenix) -->
     <string name="juno_onboarding_default_browser_title">Make %s your go-to browser</string>
+    <!-- Title for set firefox as default browser screen.
+        Note: The word 'Firefox' should not be translated -->
```

Comment:
💡 Maybe we should add a note that this is used with Nimbus and that doesn't allow usage of arguments